### PR TITLE
Add information about supported python versions

### PR DESCRIPTION
**Pull request by drknzz**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -40,7 +40,7 @@ body:
     id: python-version
     attributes:
       label: Python version
-      description: What version of Python are you using?
+      description: What version of Python are you using? Note that the currently supported Python versions are 3.8 and 3.9.
```
Comment (war-in) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:43]: What do you think about adding two checkboxes?
- [ ] 3.8...
- [ ] 3.9...

Comment (drknzz) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:43]: I thought about restricting the choice, maybe to a dropdown list.
But the problem with this approach is that it does not specify which specific python version is being used, which might be crucial info when reproducing some bugs.
We might list every python version for 3.8 / 3.9 though 🙈

Comment (war-in) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:43]: I don't think listing every version is a good idea :/ Let's maybe move the new sentence under the `What version of Python are you using?`

Comment (drknzz) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:43]: I don't think it's possible in these forms

**Review by war-in — APPROVED** [review]

**Pull request merged**